src/main/java/org/jabref/gui/groups/GroupTreeViewModel.java

```diff
@@ -147,13 +147,15 @@ private void onActiveDatabaseChanged(Optional<BibDatabaseContext> newDatabase) {
     /**
      * Opens "New Group Dialog" and add the resulting group to the specified group
      */
-    public void addNewSubgroup(GroupNodeViewModel parent) {
+
+    public void addNewSubgroup(GroupNodeViewModel parent, boolean isGroup) {
```

Comment:
Code style wise you should avoid boolean "flags", they can get confusing. It's better to introduce an enum with the two possible options. That makes it easier to recognize the purpose of the parameter.